src/journal/bsod.c

```diff
@@ -188,11 +204,20 @@ static int display_emergency_message_fullscreen(const char *message) {
         if (r < 0)
                 return log_warning_errno(r, "Failed to write emergency message to terminal: %m");
 
+        if (message_id) {
+                char buff[STRLEN("\nScan the QR code below, and search for the entry with the ID: --") + strlen(message_id) + 1];
+                xsprintf(buff, "\nScan the QR code below, and search for the entry with the ID: --%s", message_id);
+
+                r = loop_write(fd, buff, SIZE_MAX, /* do_poll = */false);
+                if (r < 0)
+                        return log_warning_errno(r, "Failed to write emergency message to terminal: %m");
+        }
+
         r = fdopen_independent(fd, "r+", &stream);
         if (r < 0)
                 return log_error_errno(errno, "Failed to open output file: %m");
 
-        r = print_qrcode_full(stream, "Scan the QR code", message, qr_code_start_row, qr_code_start_column, w.ws_col, w.ws_row);
+        r = print_qrcode_full(stream, "Scan the QR code", "https://cgit.freedesktop.org/systemd/systemd/plain/catalog/systemd.catalog", qr_code_start_row, qr_code_start_column, w.ws_col, w.ws_row);
```

Comment:
The URL seems to be outdated. @bluca Where is the latest URL?

BTW, please wrap such long line.